.circleci/config/@config.yml

```diff
@@ -102,3 +102,14 @@ workflows:
           requires:
             - publish
           <<: *master_branch_only_filter
+
+  NIGHTLY:
```

Comment:
what's the reasoning of doing it nightly and not only as PR-checks and on merges?

Reply:
Because we need exclusivity of the OpenShift cluster... if this runs on merge to master and then someone merges to develop, we have a problem

Reply:
oh so we don't use any circleci openshift images? we have a single cluster where all the tests run on?

Reply:
We can't spin a CircleCI VM because OS4 needs virtualization (and CircleCI doesn't support nested virtualization) so we end up using the same GCP test cluster like the one for merged PRs